data/phi0119/phi006/__cts__.xml

```diff
@@ -1,11 +1,11 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:phi0119.phi006" xml:lang="lat" groupUrn="urn:cts:latinLit:phi0119">
             <ti:title xml:lang="lat">Casina</ti:title>
             <ti:translation xml:lang="eng" urn="urn:cts:latinLit:phi0119.phi006.perseus-eng2" workUrn="urn:cts:latinLit:phi0119.phi006">
-              <ti:label xml:lang="eng">Casina, or The Stratagem Defeated</ti:label>
-              <ti:description xml:lang="eng">Plautus, Titus Maccius, creator; Riley, H. T. (Henry Thomas), 1816-1878, translator</ti:description>
+              <ti:label xml:lang="mul">Casina, or The Stratagem Defeated</ti:label>
+              <ti:description xml:lang="eng">Plautus Riley, H. T. , translator</ti:description>
             </ti:translation>
           <ti:edition urn="urn:cts:latinLit:phi0119.phi006.perseus-lat2" workUrn="urn:cts:latinLit:phi0119.phi006">
               <ti:label xml:lang="lat">Casina</ti:label>
-              <ti:description xml:lang="eng">Plautus, Titus Maccius, creator; Leo, Friedrich, 1851-1914, editor</ti:description>
+              <ti:description xml:lang="eng">Plautus Leo, Friedrich,Riley, H. T. editor</ti:description>
```

Comment:
unexpected space and two names merged?

Reply:
Is this Friedrich or Riley? Looks like the find/replace added Riley here?

Reply:
hey @lcerrato I thought I had caught that but like I said this second set of changes I had just started and was really only wanting to have the titles reviewed here.